src/Microsoft.DocAsCode.Build.JavaScriptReference/BuildInputs/MemberType.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.JavaScriptReference
+{
+    using System;
+
+    [Serializable]
+    public enum MemberType
+    {
+        Default,
+        Toc,
+        Package,
+        Class,
+        Interface,
+        Enum,
```

Comment:
It's supported in JSDoc. see: [http://usejsdoc.org/tags-enum.html](http://usejsdoc.org/tags-enum.html)

---
In reply to: [93179738](https://github.com/dotnet/docfx/pull/1016#discussion_r93179738) [](ancestors = 93179738)